.docs/user-guide/troubleshooting.md

```diff
@@ -0,0 +1,307 @@
+# Troubleshooting
+
+It's not doing what I expected...
+
+---
+
+## Configuration Validation
+REX-Ray configuration will usually be set in the yaml file located 
+at `/etc/rexray/config.yml`
+
+If you are having issues, or anytime you edit your configuration, you may wish 
+to validate the result by cut and paste to an online validator such as this 
+one (if your configuration contains sensitive security credentials install and use a local validator instead): 
```

Comment:
Hi @cantbewong,

Normally I'd just ignore this one line that's greater than 80 characters long since you did such a fantastic job of refactoring the file like I asked, but since I'm going to ask you to squash your commits anyway, please go ahead and take care of this line too. Thanks!